extensions/poetry/PhylumExt.toml

```
@@ -3,8 +3,8 @@ description = "Poetry package manager hooks"
 entry_point = "main.ts"
 
 [permissions]
-run = ["./", "/bin", "/usr/bin", "~/.pyenv", "~/.local/bin/poetry", "~/Library/Application Support/pypoetry", "~/.local/share/pypoetry"]
-write = ["./", "~/.cache/pypoetry", "~/Library/Caches/pypoetry", "~/.pyenv"]
+run = ["./", "/bin", "/usr/bin", "~/.pyenv", "~/.local/bin/poetry", "~/Library/Application Support/pypoetry", "~/.local/share/pypoetry", "~/.local/pipx"]
+write = ["~/.cache/pypoetry", "~/Library/Caches/pypoetry", "~/.pyenv"]
```

```suggestion
write = ["./", "~/.cache/pypoetry", "~/Library/Caches/pypoetry", "~/.pyenv"]
```
It turns out the current directory _is_ needed...for the case when no Python virtual environment, as managed by `poetry`, already exists. At least, it is needed when the [`virtualenvs.in-project` setting](https://python-poetry.org/docs/configuration#virtualenvsin-project) is `true`...which is _very common_.